pages/1.11/networking/SDN/dcos-overlay/index.md

```diff
@@ -0,0 +1,350 @@
+
+- For intra-node IP discovery we use an overlay orchestrator called Virtual Network Service. This operator-facing system component is responsible for programming the overlay backend using a library called [lashup](https://github.com/dcos/lashup) that implements a gossip protocol to disseminate and coordinate overlay routing information among all Mesos agents in the DC/OS cluster.
+
+**Note:** Your network must adhere to the [DC/OS system requirements](/1.11/installing/oss/custom/system-requirements/) to use DC/OS Overlay.
+
+
+# Adding virtual networks during installation
+
+DC/OS virtual networks can only be added and configured at install time. To replace or add another virtual network, [reinstall DC/OS according to these instructions](#replace).
+
+You can override the default network or add additional virtual networks by modifying your `config.yaml` file:
+
+```yaml
+    agent_list:
+    - 10.10.0.117
+    - 10.10.0.116
+    # Use this bootstrap_url value unless you have moved the DC/OS installer assets.
+    bootstrap_url: file:///opt/dcos_install_tmp
+    cluster_name: &lt;cluster-name&gt;
+    master_discovery: static
+    master_list:
+    - 10.10.0.120
+    - 10.10.0.119
+    - 10.10.0.118
+    resolvers:
+    # You probably do not want to use these values since they point to public DNS servers.
+    # Instead use values that are more specific to your particular infrastructure.
+    - 8.8.4.4
+    - 8.8.8.8
+    ssh_port: 22
+    ssh_user: centos
+    dcos_overlay_network:
+      vtep_subnet: 44.128.0.0/20
+      vtep_mac_oui: 70:B3:D5:00:00:00
+      overlays:
+        - name: dcos
+          subnet: 9.0.0.0/8
+          prefix: 26
+        - name: dcos-1
+          subnet: 192.168.0.0/16
+          prefix: 24
+```
+
+In the above example, we have defined two virtual networks. The virtual network `dcos` retains the default virtual network, and we have added another virtual network called `dcos-1` with subnet range `192.168.0.0/16`. When you create a network, you must give it a name and a subnet. That name is used to launch Marathon tasks and other Mesos framework tasks using this specific virtual network. Due to restrictions on the size of Linux device names, the virtual network name must be less than thirteen characters. Consult the [limitations](#limitations) section of the main Virtual Networks page to learn more.
+
+# Retrieving virtual network state
+
+After DC/OS installation is complete, you can query the virtual network configuration using the `https://leader.mesos:5050/overlay-master/state` endpoint from your browser. The `network` key at the bottom lists the current overlay configuration and the `agents` key is a list showing how overlays are split across the Mesos agents. The following shows the network state when there is a single overlay in the cluster named `dcos`.
```

Comment:
s/endpoint from your browser/endpoint/ - I'm doubtful that `leader.mesos` is available by browser in most situations

Reply:
+1